suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -955,6 +955,13 @@ export const accountSearchFn = (
     const metadataMatch = !!metadataAccountLabel?.toLowerCase().includes(searchString);
     const accountLabelMatch = !!account.accountLabel?.toLowerCase().includes(searchString);
 
+    const tokenFilterFn = (token: TokenInfo) =>
```

Comment:
nit: We usually use a verb in functions naming. For example 

filterTokensBySearch
matchesTokenSearch
isTokenMatchingSearch
doesTokenMatchSearch
filterTokenByQuery

I think it's not necessary to use "Fn" in naming, rather provide more info on what the function **does**